scripts/build-u-boot-rb1.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+
+set -eu
+
+# patches in this repo/branch were submitted upstream here:
+# https://lore.kernel.org/u-boot/20250410080027.208674-1-sumit.garg@kernel.org/
+GIT_REPO="https://github.com/b49020/u-boot"
```

Comment:
Do we need a weekly job from a tree that is not upstream / moving?

Reply:
Good point; I guess we could click the workflow run button whenever we need; having some regular build jobs keeps us honest that the script works, but I guess it's a bit overkill, happy to drop the cron.

Reply:
Dropped https://github.com/qualcomm-linux/qcom-deb-images/compare/a4e580f3e4c3a2337849bea513c52f08246b60fd..c28c68c7b28c85c4d4c01b58cd3c630010961693#diff-0439cce7d0ca8a0fe6eaf232e2a5f2e695ef9036a03378edbf2b4aa39531a638